Tests/MakerTests.swift

```diff
@@ -382,7 +438,7 @@ class MakerTests: BaseTest {
         containet.addSubview(view2)
         
         containet.configureFrame { maker in
```

Comment:
Is `containet` a correct name of this variable? Or it is a typo?

Reply:
🤔Think there's should be `contained`

Reply:
Or better `containerView`